.rubocop.yml

```diff
@@ -0,0 +1,59 @@
+inherit_from:
+  - .rubocop_todo.yml
+
+AllCops:
+  DisplayCopNames: true
+  DisplayStyleGuide: true
+  Exclude:
+    - 'lib/generators/**/*'
+    - 'vendor/**/*'
+    - 'spec/dummy/**/*'
```

Comment:
I think the dummy app should be checked by Robucop, and the failure in the CI added to `.rubocop_todo.yml`